static/src/js/pages/KeywordManagerPage/KeywordManagerPage.jsx

```diff
@@ -1,25 +1,44 @@
 import React, {
-  Suspense,
   useState,
   useCallback,
   useEffect
 } from 'react'
 import { FaPlus } from 'react-icons/fa'
+import {
+  Row,
+  Col,
+  Button
+} from 'react-bootstrap'
+import PropTypes from 'prop-types'
 import Modal from 'react-bootstrap/Modal'
-import Button from 'react-bootstrap/Button'
 
 import ErrorBoundary from '@/js/components/ErrorBoundary/ErrorBoundary'
-import LoadingTable from '@/js/components/LoadingTable/LoadingTable'
+import KeywordForm from '@/js/components/KeywordForm/KeywordForm'
 import Page from '@/js/components/Page/Page'
 import PageHeader from '@/js/components/PageHeader/PageHeader'
 import KmsConceptVersionSelector from '@/js/components/KmsConceptVersionSelector/KmsConceptVersionSelector'
 import KmsConceptSchemeSelector from '@/js/components/KmsConceptSchemeSelector/KmsConceptSchemeSelector'
+
+import parseRdfDataToInitialData from '@/js/utils/parseRdfDatatoInitialData'
```

Comment:
maybe calll parseRdfDataToFormData (as you are turning it into form data in json schema terms).   or createFormDataFromRdf